pkg/release/prerequisites.go

```diff
@@ -80,10 +83,26 @@ func (*defaultPrerequisitesChecker) DockerVersion() (string, error) {
 	return res.OutputTrimNL(), err
 }
 
-func (p *PrerequisitesChecker) Run() error {
+func (*defaultPrerequisitesChecker) DiskSpaceAvailable(
```

Comment:
why we dont use something like: https://github.com/shirou/gopsutil/

Reply:
Makes sense, changed the implementation to use the new dependency. :+1:

Reply:
cool